make.sh

```diff
@@ -62,7 +62,7 @@ clean_command="rm -rf .waf-*-* ; ./waf clean > /dev/null ; rm -rf .build > /dev/
 reset_axioms_command="PYTHONPATH=/volumes/src ufora/scripts/resetAxiomSearchFunction.py"
 rebuild_axioms_command="PYTHONPATH=/volumes/src ufora/scripts/rebuildAxiomSearchFunction.py"
 configure_command="CC=clang-3.5 CXX=clang++-3.5 ./waf configure"
-build_once_command="CC=clang-3.5 CXX=clang++-3.5 ./waf install"
+build_once_command="CC=clang-3.5 CXX=clang++-3.5 ./waf install -vv"
 
```

Comment:
This might be the reason why the build times out -vv produces *a lot* of output, you may want to revert back this change